### PR TITLE
Guess more reasonable numbers for 9782

### PR DESCRIPTION
**Pull request by mcm001**

### CODE REVIEW

File: photon-core/src/main/java/org/photonvision/common/util/TimedTaskManager.java [hunk 1]
```diff
@@ -45,8 +45,34 @@ public Thread newThread(@NotNull Runnable r) {
         }
     }
 
-    private final ScheduledExecutorService timedTaskExecutorPool =
-            new ScheduledThreadPoolExecutor(2, new CaughtThreadFactory());
+    private final ScheduledThreadPoolExecutor timedTaskExecutorPool =
```
Comment (Juniormunk) [inline photon-core/src/main/java/org/photonvision/common/util/TimedTaskManager.java:48]: What does this change do?

File: photon-core/src/main/java/org/photonvision/vision/processes/VisionModule.java [hunk 2]
```diff
@@ -451,6 +451,10 @@ boolean setPipeline(int index) {
         } else {
             pipelineSettings.cameraRedGain = -1;
             pipelineSettings.cameraBlueGain = -1;
+
+            // All other cameras (than picams) should support AWB temp
```
Comment (Juniormunk) [inline photon-core/src/main/java/org/photonvision/vision/processes/VisionModule.java:455]: It looks like the ov9281 "supports" awb but it doesn't do anything. Do we want to do anything about that case? Just leave it?

File: photon-core/src/main/java/org/photonvision/common/util/TimedTaskManager.java (hunk 1 above)
Comment (Juniormunk) [reply photon-core/src/main/java/org/photonvision/common/util/TimedTaskManager.java:48]: kinda looks like this is causing the builds to fail

**Review by Juniormunk — CHANGES_REQUESTED** [review]

File: photon-core/src/main/java/org/photonvision/common/util/TimedTaskManager.java (hunk 1 above)
Comment (mcm001) [reply photon-core/src/main/java/org/photonvision/common/util/TimedTaskManager.java:48]: It's so I can get stack traces from timed tasks that throw an exception. We could instead wrap every lambda we get in our own try catch that might be less cursed?

File: photon-core/src/main/java/org/photonvision/vision/processes/VisionModule.java (hunk 2 above)
Comment (Juniormunk) [reply photon-core/src/main/java/org/photonvision/vision/processes/VisionModule.java:455]: Meh. Yolo until someone complains.

**Review by Juniormunk — APPROVED** [review]
🚢

**Comment by mcm001** [comment]
Actually seems to make my 2311 behave odd

File: photon-core/src/main/java/org/photonvision/vision/camera/LibcameraGpuSettables.java [hunk 3]
```diff
@@ -260,4 +260,24 @@ public double getMinExposureRaw() {
...
         return this.maxExposure;
     }
+
+    @Override
+    public void setAutoWhiteBalance(boolean autowb) {
+        throw new RuntimeException("Libcamera doesn't support AWB temp (yet)");
+    }
+
+    @Override
+    public void setWhiteBalanceTemp(double temp) {
```
Comment (mcm001) [inline photon-core/src/main/java/org/photonvision/vision/camera/LibcameraGpuSettables.java:270]: @Juniormunk can you try this on your pi or something?

Comment (mcm001) [reply photon-core/src/main/java/org/photonvision/vision/camera/LibcameraGpuSettables.java:270]: Who uses picam amirite

**Pull request merged**